.vscode/settings.json

```diff
@@ -1,6 +1,68 @@
 {
     "files.associations": {
-        "*.tcc": "cpp",
-        "iomanip": "cpp"
-    }
+			"*.tcc": "cpp",
```

Comment:
This file should not be committed.

Reply:
Fixed in 2224c80a4cbaf67e853a68ca9fd92ec0c306a319